process/block/gasConsumption.go

```diff
@@ -0,0 +1,173 @@
+package block
+
+import (
+	"sync"
+
+	"github.com/multiversx/mx-chain-core-go/core/check"
+	"github.com/multiversx/mx-chain-core-go/data"
+	"github.com/multiversx/mx-chain-go/process"
+	"github.com/multiversx/mx-chain-go/sharding"
+)
+
+const (
+	initialLimitsFactor = uint64(100)
```

Comment:
maybe this one we will set to 200% for block gas overestimation (to compensate tx gas limit overestimations)